### ISSUE TITLE
null deference, not check malloc result

### ISSUE THREAD

**Issue by jmp0x7c00** [intel/intel-sgx-ssl#123]
in file `sgx/test_app/enclave/TestEnclave.cpp`:
```C++
// private key - string
	len = i2d_PrivateKey(evp_pkey, NULL);
	buf = (unsigned char *) malloc (len + 1); // not check NULL
	tbuf = buf;
	i2d_PrivateKey(evp_pkey, &tbuf);
```

**Comment by jinghe-INTC** [comment]
Will be fixed in the latest [PR](https://github.com/intel/intel-sgx-ssl/pull/141)

**Comment by jinghe-INTC** [comment]
The bug was fixed in commit 71e84cdff060d8f4c7abf08aed5c078adc90f910.
Close this issue now. Thanks!
